### DISCUSSION TITLE
Enabling plugins

### DISCUSSION THREAD

**Discussion by marcin-przywoski** [tcgoetz/GarminDB#152] [Q&A] [answered]
How do I enable a plugin? Is there a specific place I need to put the scripts in? Do I need to change something in the config file? I'll be grateful for any answers :)

**Comment by tcgoetz** [comment, marked as answer]
Documentation about plugins starts here https://github.com/tcgoetz/GarminDB#plugins and links to here https://github.com/tcgoetz/GarminDbPlugins
